docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,20 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Mirroring and rewriting the docker.io registry
```

Comment:
Maybe "Normalization of references to docker.io"? Maybe I need more coffee though :^)

Reply:
Sure, change it to `Normalization of docker.io references`